VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/join-private-consultation/join-private-consultation.component.ts

```diff
@@ -95,7 +96,10 @@ export class JoinPrivateConsultationComponent {
     }
 
     getParticipantHearingRoleText(participant: ParticipantResponse) {
-        return participant.representee ? `${participant.hearing_role} for ${participant.representee}` : participant.hearing_role;
+        const translatedtext = this.translateService.instant('join-private-consultation.for');
+        return participant.representee
+            ? `${participant.hearing_role} ${translatedtext} ${participant.representee}`
```

Comment:
hearing role can be translated
this.translateService.instant(
'hearing-role.'+participant.hearing_role)

Reply:
will also need to replace ' ', '-' and tolower

Reply:
for id